### ISSUE TITLE
v6 Set tab - functional problem

### ISSUE THREAD

**Issue by iv-gha** [thebigg73/OpenSongTablet#259]
Hello Gareth,

The set  highlights / list scroll  are being considered for local loads but not for received songs.  Perhaps a need to ensure Set tab and Song tab updated are called as part of song load.

Cheers
Ian

**Comment by iv-gha** [comment]
?

**Comment by thebigg73** [comment]
I'm assuming this is because received songs have a different file location to the song menu and set menu items.  Both have a folder/filename identification, whereas received songs are temporarily stored in ../Received/filename, so can't (and probably shouldn't) be identified as the same item.  Have I understood correctly?

**Comment by iv-gha** [comment]
If it is in the list, highlight and scroll.  If not, clear highlight, scroll to top.  A received song is not in the list, so 'if not' action applies.  This is one of the 'on song load' activities.

**Comment by iv-gha** [comment]
?

**Comment by thebigg73** [comment]
Should be addressed in updates.  Set menu is scrolled to position when song is loaded (in the do after settle section).  If not found, the set scrolls to the top - I might change this part, as sometimes it is better to keep the current last used set position.  This would allow a user to deviate from a set, but quickly resume from their planned order?

**Comment by iv-gha** [comment]
I think highlight means the current song - it is confusing that it may mean something else.

**Comment by thebigg73** [comment]
Correct.   Highlight is the current song.  So if not in a set, no song is
highlighted in the set.  I just meant not to scroll to the top if the song
being viewed is not in the set, just unhighlight any previously
selected/highlighted item
